org.electronjs.Electron2.BaseApp.yml

```diff
@@ -85,3 +84,25 @@ modules:
         x-checker-data:
           type: git
           tag-pattern: ^v([\d.]+)$
+
+  - name: setup-separate-locales
+    buildsystem: simple
+    build-commands:
+      - install -m755 setup-separate-locales /app/bin/setup-separate-locales
+    sources:
+      - type: script
+        dest-filename: setup-separate-locales
+        commands:
+          - |
+            APP_LOCALES_DIR=$1/locales
+            RUNTIME_LOCALES_DIR=${FLATPAK_DEST}/share/runtime/locale
+            if [ -d ${APP_LOCALES_DIR} ]
+            then
+                for lang in ${APP_LOCALES_DIR}/*.pak
+                do
+                locale="$(basename -s .pak $lang)"
+                install -Dm644 -t "${RUNTIME_LOCALES_DIR}/${locale%%-*}/" "$lang"
```

Comment:
nit: could save the first path into an intermediate variable to avoid repeating it